mdaftarirp/aws-bedrock-with-rag-and-react
# Running on Containers / Infrastructure as Code (CDK)

## Summary
This PR simplifies the solution deployment process by using CDK for deployment and now leverages AWS App Runner for the backend and S3 / CloudFront for the frontend. It streamlines the deployment workflow and reduces infrastructure complexity.

## Implementation Details
- `BackendStack`: Now uses App Runner alpha module to deploy the backend service.
- `FrontendStack`: Simplified to use default CloudFront URL without custom domain configuration.
- Both stacks now output their respective URLs for easy access.

## Testing Instructions
1. Ensure you have the latest CDK CLI installed.
2. Run `cdk bootstrap` if you haven't already done so in your account/region.
3. Deploy the backend: `cdk deploy BackendStack`
4. Deploy the frontend: `cdk deploy FrontendStack`
5. Use the outputted URLs to verify that both the backend and frontend are working correctly.
